### PR TITLE
feat(across): add info modal and minor fixes

### PR DESCRIPTION
**Pull request by Gamaranto**
**Motivation**
This PR adds the information modal and is bundled with 2 additional small UI improvements, it makes the footers on the Send and Pool tab sticky again and it add the "ticket" border to the accent section like in the designs

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [x]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: src/components/SendAction/SendAction.styles.ts [hunk 1]
```diff
@@ -1,9 +1,32 @@
...
+  }
+`;
 
 export const Wrapper = styled.div`
+  position: relative;
   display: flex;
   flex-direction: column;
-  padding: 40px 0;
+  padding: 65px 0;
 `;
```
Comment (Tulun) [inline src/components/SendAction/SendAction.styles.ts:30]: Should this be in rems?

**Review by Tulun — APPROVED** [review]
Minor nit, otherwise LGTM!

File: src/components/InformationDialog/InformationDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+          dependent on Ethereum Gas fees. View here to learn more about Ethereum
+          gas fees.
+        </Text>
+      </Info>
+
+      <List>
+        <li>
+          Click{" "}
+          <Link
+            href="https://app.gitbook.com/o/-MlBqw0sVAJZO0pl5jGc/s/-MlBr3oaDzSVL5ybJHcV/how-does-across-work-1/architecture-process-walkthrough"
```
Comment (daywiss) [inline src/components/InformationDialog/InformationDialog.tsx:53]: this takes me to a site where i need to log in, i dont think its right

**Review by daywiss — CHANGES_REQUESTED** [review]
looks good just need to fix that link. also not sure if this got linted, i added a `yarn lint` script

File: src/components/SendAction/SendAction.styles.ts (hunk 1 above)
Comment (Gamaranto) [reply src/components/SendAction/SendAction.styles.ts:30]: i think in general px values are better for spacing as they don't scale up with the user increasing their base font

File: src/components/InformationDialog/InformationDialog.tsx (hunk 2 above)
Comment (Gamaranto) [reply src/components/InformationDialog/InformationDialog.tsx:53]: great catch!!

**Review by daywiss — APPROVED** [review]

**Pull request merged**